Makefile

```diff
@@ -209,3 +209,6 @@ image-pruner-setup:
 	oc adm policy --as=system:admin add-cluster-role-to-user system:image-pruner -z image-pruner
 	$(MAKE) apply WHAT=cluster/ci/jobs/image-pruner.yaml
 .PHONY: image-pruner-setup
+
+cluster-operator-roles:
```

Comment:
Where is your role?

Reply:
Your role and the definition belongs in this repo if you are asking for permissions.

Reply:
The role lives with the app and all the ansible we use to deploy it: https://github.com/openshift/cluster-operator/blob/master/contrib/examples/cluster-operator-roles-template.yaml

My thought was it's pretty easy for us to upload the template with our ansible, as opposed to maintaining two copies of the same data in different repos. 

Is that however required for review purposes?